content/02.main-text.md

```diff
@@ -120,9 +120,11 @@ For now however, the primary Manubot output is HTML intended to be viewed in a w
 
 Manubot performs continuous publication: every update to a manuscript's source is automatically reflected in the online outputs.
 The approach uses continuous integration (CI) [@doi:10.1038/550143a; @doi:10.1038/nbt.3780; @doi:10.1101/344804], specifically [Travis CI](https://travis-ci.org/) at the moment, to monitor for changes.
-When changes occur, the CI service generates an updated manuscript.
+When changes occur, the CI service attempts to generate an updated manuscript.
 If this process is error free, the CI service timestamps the manuscript and uploads the output files to the GitHub repository.
 Since the HTML manuscript is hosted using [GitHub Pages](https://pages.github.com/), the CI service automatically deploys the new manuscript version when it pushes the updated outputs to GitHub.
+Use of CI to build the manuscript allows users to catch common errors (e.g. tag typos, non-existent tags, or JSON format errors) as well as detect broken or missing citations.
```

Comment:
How about changing `allows users to catch` to `automatically catches`?

Tag may not be understood to refer to citations.  Can we say `citation tag typos`?

I think you can remove `detect` in the last phrase.

Reply:
I am in favor of being more general on the types of errors CI catches:

> such as misspelled citations, invalid formatting, or misconfigured software dependencies.